contracts/src/v0.8/tests/StreamsLookupUpkeep.sol

```diff
@@ -88,6 +88,14 @@ contract StreamsLookupUpkeep is AutomationCompatibleInterface, StreamsLookupComp
     return (callbackReturnBool, performData);
   }
 
+  function checkErrorHandler(
```

Comment:
This is testcode for automation, right? Can we move this to the automation folder?

Reply:
yes, will do that in AUTO-9097. I don't want to change this PR since this is a cherry pick from a release fix and i want to merge as is